src/query/ast/src/parser/statement.rs

```diff
@@ -1959,6 +1992,23 @@ pub fn priv_type(i: Input) -> IResult<UserPrivilegeType> {
     ))(i)
 }
 
+pub fn stage_priv_type(i: Input) -> IResult<UserPrivilegeType> {
```

Comment:
just READ | WRITE | USAGE is ok?

Reply:
After discussion, now the privileges of the stage are confirmed.

```
read
write
```